<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionOptions</class>
 <widget class="QDialog" name="SessionOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/ico/conf.png</normaloff>:/resource/ico/conf.png</iconset>
  </property>
  <widget class="QTreeWidget" name="treeWdtOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>271</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="autoExpandDelay">
    <number>-1</number>
   </property>
   <property name="rootIsDecorated">
    <bool>true</bool>
   </property>
   <property name="uniformRowHeights">
    <bool>false</bool>
   </property>
   <property name="itemsExpandable">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="animated">
    <bool>false</bool>
   </property>
   <property name="allColumnsShowFocus">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Options</string>
    </property>
    <property name="textAlignment">
     <set>AlignHCenter|AlignVCenter|AlignCenter</set>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Connection</string>
    </property>
    <item>
     <property name="text">
      <string>Serial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NetWork</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Terminal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Language</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelTital">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(96, 157, 255);
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpBoxSerial">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>22</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="cmbBoxPort">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBoxBaudRate">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBoxDataBits"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBoxParity"/>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBoxStopBits"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>22</y>
      <width>68</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBaudRate">
       <property name="text">
        <string>Baud rate：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDataBits">
       <property name="text">
        <string>Data bits：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelParity">
       <property name="text">
        <string>Parity：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelStopBits">
       <property name="text">
        <string>Stop bits：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groBoxNET">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>59</y>
      <width>225</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelPort_IP">
       <property name="text">
        <string>NetWork Port：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groBoxLanguage">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget4">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>56</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButEN">
       <property name="text">
        <string>English</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButCN">
       <property name="text">
        <string>中文</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelWarning">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Warning:  Restart to take effect</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelMsg">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>282</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groBoxTerminal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
